app/src/main/java/campus/tech/kakao/contacts/AddInfo.kt

```diff
@@ -0,0 +1,124 @@
+package campus.tech.kakao.contacts
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.ImageView
+import android.widget.TextView
+import android.widget.Toast
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import kotlin.collections.ArrayList
+
+class AddInfo : AppCompatActivity() {
```

Comment:
Activity 등의 특별한 목적을 가지는 클래스의 경우 Activity suffix 를 붙이는 것이 조금 더 보기 편합니다! 

```suggestion
class AddInfoActivity : AppCompatActivity() {
```